devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -132,6 +138,8 @@ def levelEventHandler(currentLevel) {
 	if (lastLevel == "undefined" || currentLevel == lastLevel) { //Ignore invalid reports
 		log.debug "Ignore invalid reports"
 	} else {
+		state.invalidSameLevelEvent = true
```

Comment:
Please, could You explain why did You set this state variable and never use it in the code ?
(I think this one is used in  another DTH only zigbee-window-shade.groovy)

Reply:
Yes, It's a another PR problem, I'll remove the code.